### PR TITLE
Berry Picking, java sol

### PR DESCRIPTION
**Pull request by HarryWangATX**

### CODE REVIEW

File: solutions/USACO jan20-berry-picking.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                count += B[j]/i;
+                leftOver[j] = B[j]%i;
+            }
+
+            if(count >= K){
+                //If count can account for both Bessie and Elsie, then we can just allocate K/2 * i to Bessie
+                res = Math.max(res, K/2 * i);
+            }
+            else if(count >= K/2){
+                //If count can only account of Elsie and some of Bessie, then we can take from the left overs.
```
Comment (frodakcin) [inline solutions/USACO jan20-berry-picking.mdx:48]: Typo "account of Elsie" and "left overs"

Probably should be
`//If count can only account for Elsie and some of Bessie, then we can take from the leftovers.`

**Review by frodakcin — CHANGES_REQUESTED** [review]
All good, just some minor typos.

**Review by frodakcin — APPROVED** [review]
All set.

File: solutions/USACO jan20-berry-picking.mdx [hunk 2]
```diff
@@ -0,0 +1,139 @@
+---
+id: usaco-990
+source: USACO Silver 2020 January
+title: Berry Picking
+author: Qi Wang
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_berries_silver_jan20.html)
+
+**Time Complexity:** $\mathcal{O}(max(B_i) \cdot N\log N)$
```
Comment (dolphingarlic) [inline solutions/USACO jan20-berry-picking.mdx:10]:
```suggestion
**Time Complexity:** $\mathcal{O}(\max(B_i) \cdot N\log N)$
```

**Review by dolphingarlic — COMMENTED** [review]
Just one more minor thing

**Pull request merged**